api/include/mfxdefs.h

```diff
@@ -21,7 +21,7 @@
 #define __MFXDEFS_H__
 
 #define MFX_VERSION_MAJOR 1
-#define MFX_VERSION_MINOR 28
+#define MFX_VERSION_MINOR 29
```

Comment:
Is it ok to advance API version with undocumented changes?

Reply:
Fixed